index.d.ts

```diff
@@ -0,0 +1,24 @@
+import { GraphQLSchema } from 'graphql'
+import { FastifyInstance } from 'fastify'
+import { MercuriusOptions } from 'mercurius'
+
+interface buildFederationSchemaOptions {
```

Comment:
nit: I always recommend exporting interfaces & types if possible as you never know when you need them!

Reply:
done